package.json

```diff
@@ -19,18 +19,21 @@
     "@types/jest": "^26.0.23",
     "@types/node": "^15.3.0",
     "@types/node-fetch": "^2.5.10",
+    "babel-eslint": "^10.1.0",
```

Comment:
Did you add this dependency? 🤔

Reply:
Interesting. I do not think so. In terms of linting I checked package json for a linting script and I do remember running `format:validate`.